nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -151,11 +151,13 @@ in
       };
 
       daemonIONiceLevel = mkOption {
-        type = types.int;
+        type = types.either types.int (types.enum ["idle"]);
```

Comment:
It feels like cramming this "idle" setting in here makes the user interface weird and counter-intuitive. Personally, I'd rather have an additional setting for this purpose which accepts a boolean to enable/disable "idle".